### DISCUSSION TITLE
Chat UI suggestion

### DISCUSSION THREAD

**Discussion by TheHung184** [balzack/databag#84] [Ideas]
Hi @balzack , I really like the color and design of Databag. I also would love to see chat UI with alternative shade for each person in chat. That would be easier to get who is talking previously. I know avatar can do that, but it is small and sometimes they can be pretty similar. Another idea i think would be great is the ability to reply to one message.
Thank you for your great tool.

**Comment by balzack** [comment]
I really like the 'alternate shade for each person in the chat' idea! Do you see it as something configurable, or an auto/random shade selection for each person in the chat?

I'll add the 'reply to one message' to the backlog also. It's similar to emoji reactions, so I will bundle it with that feature.

Thanks for the suggestions!

**Reply by TheHung184** [reply]
Great idea to make it configurable ! I think it should be cool to assign shade automatically or based on one avatar if not config and follow the user choose when select in the user settings.

**Reply by CoffeBlock** [reply]
I like this idea!
